h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -3169,10 +3168,13 @@ def extract_from_twoDimTable(metricOfInterest, fieldOfInterest, takeFirst=False)
     """
 
     allFields = metricOfInterest._col_header
+    return  extract_field_from_twoDimTable(allFields, metricOfInterest.cell_values, fieldOfInterest, takeFirst=False)
+
+def extract_field_from_twoDimTable(allFields, cell_values, fieldOfInterest, takeFirst=False):
```

Comment:
indent seems off

Reply:
Yes, there is an extra space between return and extract_field..